backend/api/views.py

```diff
@@ -302,6 +304,52 @@ def archive(self, request, customer_id=None):
         CustomerDeleteResponseSerializer().validate(return_data)
         return Response(return_data, status=status.HTTP_200_OK)
 
+    @extend_schema(
```

Comment:
Also need to get rid of the other method.

Reply:
you mean the old draft_invoice?